lib/money/bank/variable_exchange.rb

```diff
@@ -58,8 +57,8 @@ class VariableExchange < Base
       # @param [RateStore] st An exchange rate store, used to persist exchange rate pairs.
```

Comment:
Since the param's now called `store`, I feel we should change this as well. 🤓
```suggestion
      # @param [RateStore] store An exchange rate store, used to persist exchange rate pairs.
```